File: tests/resources/java/hello_world/HelloWorld.java
public class HelloWorld { 
    public static void main(String[ ] args) {
        System.out.println(System.getenv("APPWRITE_FUNCTION_ID"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_NAME"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_TAG"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_TRIGGER"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_ENV_NAME"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_ENV_VERSION"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_EVENT"));
        System.out.println(System.getenv("APPWRITE_FUNCTION_EVENT_DATA"));
    }
}

File: tests/resources/kotlin/hello_world/HelloWorld.kt
fun main() {
    println(System.getenv("APPWRITE_FUNCTION_ID") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_NAME") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_TAG") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_TRIGGER") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_ENV_NAME") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_ENV_VERSION") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_EVENT") ?: "");
    println(System.getenv("APPWRITE_FUNCTION_EVENT_DATA") ?: "");
}